apps/condo/domains/common/utils/date.js

```diff
@@ -36,10 +43,65 @@ const getStartDates = (dateRaw) => {
  */
 const getCurrTimeStamp = () => Math.floor(Date.now() / 1000)
 
+/**
+ * Clear bad user input
+ * @param dateStr raw user input
+ * @returns {null|string}
+ */
+function clearDateStr (dateStr) {
+    if (isEmpty(dateStr) || !isString(dateStr)) {
+        return null
+    }
+
+    const sanitizedDateStr = dateStr
+        .replace(SYMBOLS_TO_CUT_FROM_DATES_REGEXP, '')
+        .trim()
+    if (isEmpty(sanitizedDateStr)) {
+        return null
+    }
+
+    return sanitizedDateStr
+}
+
+/**
+ * @param {string} dateStr
+ * @param {{formats?: Array<string>, strict?: boolean}} options
+ * @return {boolean}
+ */
+function isDateStrValid (dateStr, options) {
+    if (!dateStr) {
+        return false
+    }
+
+    const params = {
+        formats: undefined,
```

Comment:
why exactly this options?

Reply:
I though that if it's in common utils, it should accept any parameters. Now i moved it to specific utils and force needed params